kolibri/core/content/api.py

```diff
@@ -621,6 +621,15 @@ def map_file(file):
     return file
 
 
+# A map of fields that did not used to be in the BaseContentNodeMixin
+# but now are - the map gives a default value for these to be filled in with
+# if not present on an API request from a contentnode public API endpoint
+contentnode_previously_omitted_fields = {
```

Comment:
So, we can continue to update this map, as long as we are also updating the fields below? And we combined the previously omitted field learner needs here, and then adding the on_device_resources_field on line 657?

Reply:
Yes - this will allow us to continue adding to the contentnode public API, but make sure that any remote requests to it made using the `baseurl` query parameter will return the correct shape response even if it's an older version.